### DISCUSSION TITLE
Background processes

### DISCUSSION THREAD

**Discussion by ingdesarr1** [flet-dev/flet#2888] [Q&A]
### Question

Good morning, I am developing an application in flet and I am generating an executable using auto-py-to-exe. The problem is that when closing the executable, a process of the application remains open in the background, which generates conflicts when opening it again or when trying to delete the executable. How can I close these processes or what could be causing background processes to remain?

Thank you

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by apriolo** [comment]
@ingdesarr1 i'm with the same problem on version>20.2, when i run with command LINE or exe with pyinstaller, my python version is 3.12, im using windows 10, flet version 20.2 it worked as well, but in the recents version not, on the command line dont appears a error only dont close the script and the terminal stay frozen, but ctrl+scroll Lock Works, someone can help?
